python-api/gstswitch/helpers.py

```diff
@@ -87,7 +90,7 @@ def audio_port(self, audio_port):
     @property
     def running_tests_video(self):
         """Get the currently running test video list"""
-        return self._running_tests_video
+        return list(self._running_tests_video)
```

Comment:
Okay, I'm not that deep into the API-Semantics of python. I'll change it to return a tuple.